ethereum_history_api/circuits/lib/src/verifiers/tx_helpers/split_test.nr

```diff
@@ -9,38 +9,43 @@ mod receipt {
     use crate::verifiers::receipt::{MAX_ENCODED_RECEIPT_LENGTH, MAX_RECEIPT_RLP_LENGTH};
     use crate::receipt::TxType;
 
+    global MAX_ENCODED_LEN = 525;
+
     #[test]
     fn eip1559() {
-        let (tx_type, receipt_rlp): (TxType, [u8; MAX_RECEIPT_RLP_LENGTH]) = split_into_tx_type_and_rlp(false, encoded_receipt);
-        assert(tx_type == encoded_receipt[0], "Incorrect tx type returned");
-        assert(receipt_rlp.len() == encoded_receipt.len() - 1, "Incorrect receipt RLP length returned");
-        assert(sub_array_equals(receipt_rlp, encoded_receipt, 1), "Incorrect receipt RLP returned");
+        let encoded_receipt = Fragment::from_array(encoded_receipt);
+        let (tx_type, receipt_rlp) = split_into_tx_type_and_rlp(false, encoded_receipt);
+        assert(tx_type == encoded_receipt.at(0), "Incorrect tx type returned");
+        assert(receipt_rlp.length == encoded_receipt.length - 1, "Incorrect receipt RLP length returned");
+        assert(
```

Comment:
If you mean assert_eq_X functions from types.nr I think here is not the place for it because we don't have a natural RlpFragment here. If you mean assert_eq function, I think there's no significant difference between what I wrote and assert_eq

Reply:
I meant assert_eq. I know there is no much difference in the way it works, but it makes the code clearer